concourse/pipeline.yml

```diff
@@ -0,0 +1,38 @@
+resource_types:
+  - name: rsync-resource
+    type: docker-image
+    source: {repository: mrsixw/concourse-rsync-resource}
+
+resources:
+  - name: repository
+    type: git
+    source:
+      uri: https://github.com/spluseins/spluseins.git
+      branch: master
+
+  - name: server-deploy
+    type: rsync-resource
+    source:
+      server: spluseins.de
+      base_dir: /home/concourse/build-result
+      user: concourse
+      disable_version_path: true
```

Comment:
rein aus Neugier, was macht das?

Reply:
disable_version_path: default is false. Then false out will put content in a directory named by the version name. This directory is omitted when this option is enabled. Note that check and in origins will treat all the files in the base_dir as versions in this case.